app/Http/Controllers/AuthController.php

```diff
@@ -2,34 +2,37 @@
 
 namespace App\Http\Controllers;
 
+use App\Http\Requests\OperatorLoginRequest;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
 
-class AuthController extends Controller {
+class AuthController extends Controller
+{
 
-    public function index() {
+    public function index()
+    {
```

Comment:
I follow k&r style, so braces should not have a new line.
But this thing does not make the app broke, so we can change it back later.

Reply:
I see i follow psr-2 coding style, but if the all source in this codebase follow to k&r style i will change it.